app/javascript/controllers/report_controller.js

```diff
@@ -36,4 +47,8 @@ export default class extends Controller {
     window.history.replaceState(null, null, window.location.pathname)
     this.stimulate('Report#app_search', "")
   }
+
+  toggle_drop_down(event) {
```

Comment:
just picking the first drop down works, but I would add an id, or a data-target to the dropdown then target that just in case

Reply:
Great point.  I've added the id attribute and targeted that in the controller action.  I also gave the controller action a more specific name that corresponds to the new id.